source/change-streams.rst

```diff
@@ -339,19 +435,44 @@ Single Server Topologies
 
 Presently, change streams cannot be initiated on single server topologies as they do not have an oplog.  Drivers MUST NOT throw an exception in this scenario, but instead rely on an error returned from the server.  This allows for the server to seamlessly introduce support for this in the future, without need to make changes in driver code.
 
+startAtClusterTime
+^^^^^^^^^^^^^^^^^^
+
+:since: 4.0
+
+``startAtClusterTime`` specifies that a ``changeStream`` will only return changes that occurred after the specified timestamp.
```

Comment:
Should this be called "startAfterClusterTime", given this description?

Reply:
From https://docs.google.com/document/d/1jr_peytfjvfPasdmkar51U8UPk4-65arltQzOQ1Xsho/edit#heading=h.z4ja6bcn5490

> The drivers will need to add a 'startAtClusterTime' option to the db.watch() and db.collection.watch() helpers. To allow resuming of change streams even before the first notification, drivers should:
>  + Obtain a cluster time from the cluster. Any command (e.g. ismaster) will return the cluster time in the $clusterTime field.
>  + Start all change streams with a 'startAtClusterTime', and remember that time. 
>  + As soon as the first change (with the first resume token) is returned, they no longer have to track that time, instead remembering the resume token as they do today.
>  + If there is an error before the first change, they should resume with 'startAtClusterTime' and the original cluster time. If there is an error after the first change, they should resume with 'resumeAfter' and the resume token.